Server/registration/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM ubuntu:22.04 as base
+
+COPY wait.sh /usr/local/bin/
+
+RUN apt-get update 
+RUN apt-get install -y nginx vim curl python3  sudo git
+RUN apt-get update && apt-get install -y procps && rm -rf /var/lib/apt/lists/*
```

All the apt stuff can be on a single layer:

```suggestion
RUN apt-get update && apt-get install -y nginx vim curl python3  sudo git procps
RUN rm -rf /var/lib/apt/lists/*```

But why do we empty that directory?